executor/common.h

```diff
@@ -713,7 +713,7 @@ int main(void)
 #if SYZ_USE_TMP_DIR || SYZ_SANDBOX_ANDROID_UNTRUSTED_APP
 			use_temporary_dir();
 #endif
-			/*SANDBOX_FUNC*/
+/*SANDBOX_FUNC*/
```

Comment:
You seem to have a too old clang-format or something. Please either install a newer one, or revert these unrelated changes.